src/test/java/pageobjects/SalesforceObjects/ForecastingPage.java

```diff
@@ -44,4 +44,32 @@ public static By assignOption(String roleValue, String actionValue) {
         return By.xpath("//span//a[text()='" + roleValue + "']/parent::span/following-sibling::span//a[text()='" + actionValue + "']");
     }
 
+    public static By QuickFind= By.xpath("//input[@placeholder='Quick Find']");
```

Comment:
Please follow the consistent naming convention mentioned in the guidelines, variable name should start from small letter only